invenio_accounts/views/__init__.py

```diff
@@ -9,26 +9,7 @@
 
 """Invenio-accounts views."""
 
-from flask import abort, current_app, request
-from flask_security.views import anonymous_user_required
-from flask_security.views import login as base_login
 
-from .settings import blueprint
+from .settings import login
 
-
-@anonymous_user_required
-@blueprint.route("/login")
-def login(*args, **kwargs):
-    """Disable login credential submission if local login is disabled."""
-    local_login_enabled = current_app.config.get("ACCOUNTS_LOCAL_LOGIN_ENABLED", True)
-
-    login_form_submitted = request.method == "POST"
-    if login_form_submitted and not local_login_enabled:
-        # only allow GET requests,
-        # avoid credential submission/login via POST
-        abort(404)
-
-    return base_login(*args, **kwargs)
-
-
-__all__ = ("blueprint", "login")
```

Comment:
i don't know how, because the variable doesn't exist anymore in `settings.py`. that was the reason to move the `def login` function into `settings.py`

Reply:
and to be clear, as i understand it is also not possible to create this variable here, because i would have to call `create_settings_blueprint` here at a time where the `current_app` is not accessible

Reply:
or should i do it that way, that i move the [`blueprint = Blueprint()`](https://github.com/utnapischtim/invenio-accounts/blob/add-init-function/invenio_accounts/views/settings.py#L36-L42) outside of the `create_settings_blueprint` function?